src/content/concepts/entry-points.md

```diff
@@ -77,8 +77,79 @@ The object syntax is more verbose. However, this is the most scalable way of def
 
 T> __"Scalable webpack configurations"__ are ones that can be reused and combined with other partial configurations. This is a popular technique used to separate concerns by environment, build target, and runtime. They are then merged using specialized tools like [webpack-merge](https://github.com/survivejs/webpack-merge).
 
-T> You can pass empty object `{}` to `entry` when you have only entrypoints generated by plugins.
+T> You can pass empty object `{}` to `entry` when you have only entry points generated by plugins.
 
+### EntryDescription object
+
+An object with entry point description. You can specify the following properties.
+
+- `dependOn` :  The entry points that the current entry point depends on.They must be loaded before this entry point is loaded.
+- `filename` :  Specifies the name of each output file on disk.
+- `import`   :  Module(s) that are loaded upon startup.
+- `library`  :  Options for library.
+- `runtime`  :  The name of the runtime chunk. If set, a runtime chunk with this name is created otherwise an existing entry point is used as runtime.
```

Comment:
```suggestion
- `import`:  Module(s) that are loaded upon startup.
- `library`:  Options for library.
- `runtime`:  The name of the runtime chunk. If set, a runtime chunk with this name is created otherwise an existing entry point is used as runtime.
```